### PR TITLE
Techweb tweaks [NO GBP]

### PR DESCRIPTION
**Pull request by MTandi**
## About The Pull Request

PR for collecting techweb feedback post merge and address valid issues of #84024.

People seem to dislike the abundance of new experiments in the middle of the tree, so reducing some requirements for those for now.

Also updated NT frontier app a bit, as people are confused with the shell experiments.

image

## Why It's Good For The Game

Balancing out the new tree.

## Changelog

:cl:
balance: Added ordnance to extra access of geneticists and roboticists
balance: Reduced parts scanning tests' machine count to 4 from 8
balance: Reduced augmented organs scanning tests mob count to 1 from 2
balance: Reduced equipped mech scanning test count to 1 from 2
balance: Added polycrystal option to bluespace crystal scan test 
fix: Allowed NTNet relay in away circuit imprinter for NT Frontier app
qol: NT Frontier app installed on RD and Scientists` PDAs by default
qol: Updated NT Frontier app to be more user-friendly
/:cl:

### CODE REVIEW

**Comment by GPeckman** [comment]
Is there any chance you'd consider moving the BSRPED to an earlier node?

**Comment by MTandi** [comment]
> Is there any chance you'd consider moving the BSRPED to an earlier node?

For now just halved the machines you need to upgrade in the parts tree, to accelerate it.

## The reason for BSRPED being where it is RN
You have 3 ways of upgrading the machines ordered by convenience:

1. Manual deconstruction and replacement
2. RPED
3. BRPED

Worst one is default available roundstart.
Then you unlock RPED and you just need to open a panel and come close to upgrade, which is already a huge boost.
Then you unlock BRPED and you can do it without opening a panel and coming close.

There must be a certain window between all 3 methods for progression, so BSRPED is in a good place currently.
Although I may need to add a discount experiment if it is really blocking some gimmick and needed ASAP for someone.

Please suggest.

**Comment by RiceNoman** [comment]
Gating plasma cutters behind publishing on NT frontier feels especially harsh against any attempt to gather materials in a reasonable timeframe on lowpop.
The drill and jackhammer never get used because they're expensive materials-wise, and barely feel like an upgrade over an easily-obtained 2x cooldown PKA. If the cutter is going to be gated this far down the tech tree then the drills really need some love.

**Comment by GPeckman** [comment]
> 1. Manual deconstruction and replacement
> 2. RPED
> 3. BRPED
> 
> Worst one is default available roundstart. Then you unlock RPED and you just need to open a panel and come close to upgrade, which is already a huge boost. Then you unlock BRPED and you can do it without opening a panel and coming close.
> 
> There must be a certain window between all 3 methods for progression, so BSRPED is in a good place currently. Although I may need to add a discount experiment if it is really blocking some gimmick and needed ASAP for someone.
> 
> Please suggest.

There are a few reasons why I disagree with this. First of all, the BSRPED provides very little extra utility to the scientist themself over the normal RPED. Being able to upgrade from a distance doesn't actually help you all that much. If you're using a machine, then you already have access to it and can just walk up and RPED it. It isn't bulky and can hold more parts, but those are both relatively minor buffs, especially since things like the construction bag can already allow to carry more parts.

Where the BSRPED really shines is helping other people. Instead of having to stand around for several minutes waiting for someone to let me into chemistry or the lathe room or whatever, I can just upgrade their stuff. Scientists already rarely bother to proactively upgrade other department's machines, and making it even harder for them to do so (by making the BSRPED harder to get) it's only going to get rarer.

**Comment by Donglesplonge** [comment]
i do like the general idea behind the overhaul of the tech tree, and think its generally a good idea, but from a gene main please don't put it behind 2 required experiments, one of which isn't even tied to the geneticist itself, but xenobiology, a place they can't access, and botany, who for some reason tied into it somehow. the research already wasn't ever done (at least on manuel, the server i play on) when it was 2500 and practically roundstart, and now i don't think it'd ever get done.

**Comment by MTandi** [comment]
> Gating plasma cutters behind publishing on NT frontier feels especially harsh against any attempt to gather materials in a reasonable timeframe on lowpop. The drill and jackhammer never get used because they're expensive materials-wise, and barely feel like an upgrade over an easily-obtained 2x cooldown PKA. If the cutter is going to be gated this far down the tech tree then the drills really need some love.

I've unlocked plasma cutter on 4th minute during local testing, don't overexaggerate. And after this PR you won't need to download the app.

**Comment by MTandi** [comment]
> Where the BSRPED really shines is helping other people. Instead of having to stand around for several minutes waiting for someone to let me into chemistry or the lathe room or whatever, I can just upgrade their stuff. Scientists already rarely bother to proactively upgrade other department's machines, and making it even harder for them to do so (by making the BSRPED harder to get) it's only going to get rarer.

Do you suggest moving it to applied bluespace?

**Comment by sylvia-from-fulp-station** [comment]
> Do you suggest moving it to applied bluespace?

This would work fine and make it about inline with where it was before.

**Comment by GPeckman** [comment]
> Do you suggest moving it to applied bluespace?

Yeah, applied bluespace sounds much better. Now that I've had a chance to play with it as well, I have some more suggestions. First of all, I don't understand why advanced cybernetic organs and integrated toolsets are locked behind two ordnance experiments. I could see an argument for medical experiments, but why ordnance of all things? In addition to that, the techweb currently puts x-ray eyes earlier in the techweb than welding and flashlight eyes, which also makes very little sense. I'd move advanced organs one tier lower and swap combat implants with integrated toolsets. If you do that, it would also make sense to move the advanced tools pre-requisite to combat implants or just drop it entirely.

**Comment by mc-oofert** [comment]
Move flatpacker to the earliest unlockable node as possible (as it was intended) or i do it myself

**Comment by MTandi** [comment]
> Move flatpacker to the earliest unlockable node as possible (as it was intended) or i do it myself

It's the fastest way to build machinery, nearly effortlessly, so It makes sense that it's next to RCD.
Add them to engineering roundstart to make it a valuable piece of machinery that antags can sabotage.

**Comment by MTandi** [comment]
@Donglesplonge moved genetic scanner to bio_scan, hopefully it helps with your use case. Or maybe that second PR will fix it if it's merged.

@GPeckman @sylvia-from-fulp-station won't move BRPED earlier for now, want to observe. Feel free to create a PR if you strongly disagree.

**Comment by MTandi** [comment]
> I think mech's plasma cutter could be made a little easier, when you think about it, it should be way less advanced as there's no miniaturization involved in making a **hand-held** cutter, the mech's cutter is probably the size of a person's torso, I dunno if that'd make it messy/smth else though. The mech's PC is the most powerful, but unironically the adv cutter is more advanced considering it's only a bit less powerful in a bag-friendly format. (It's not better in gameplay, I'm mostly talking in character better)

The idea behind all this was not just to make it balanced, but also make sense.

You need to do play with cells and limbs before digging into xenobiology or even human DNA stuff.
You need to play with plasma experiments to gradually unlock high-tech plasma equipment.

The idea is that the regular plasma cutter is an engineering version intended for welding walls. It's just that miners are impatient to try this new toy against drills and PKAs. And the mech and miner variant are the proper upgrades of engi version and have a dedicated node in the mining tree.

Also mechs are quite good with drills, even without the plasma cutter.

**Comment by MTandi** [comment]
> Can you also address #84166?

Does not require addressing. Added NTNet relay for convenience to not depend on the mapspawn Console on Charlie.

File: tgui/packages/tgui/interfaces/NtosScipaper.jsx [hunk 1]
```diff
@@ -405,7 +406,7 @@ export const NtosScipaperContent = (props) => {
             })
           }
         >
-          {'View Scientific Partners'}
+          {' Scientific Partners'}
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/NtosScipaper.jsx:409]:
```suggestion
          {'Scientific Partners'}
```

**Comment by SquirrelSlapper** [comment]
Can you make it so the human autopsy and non-human autopsy are available at round start again? Currently if you do them at round start it uses up the cadavers but does not count towards the required experiments thing, making coroners more useless.

**Pull request merged**